include/aspect/particle/integrator/euler.h

```diff
@@ -38,7 +39,7 @@ namespace aspect
        * @ingroup ParticleIntegrators
        */
       template <int dim>
-      class Euler : public Interface<dim>
+      class Euler : public Interface<dim>, public SimulatorAccess<dim>
```

Comment:
I think that you don't actually need this any more now that you don't need the `initialize()` function any more.

Reply:
Same with the `#include` file.